### ISSUE TITLE
Request: Jellyfin Notify

### ISSUE THREAD

**Issue by PutzJimmy** [RandomNinjaAtk/docker-sonarr-extended#23]
Hi. Is there any plans for a JellyfinNotify.bash by any chance (Radarr/Sonarr/Lidarr). I'm still using https://hotio.dev/containers/autoscan but only for Jellyfin since your Plex script just works and is much simpler.

**Comment by RandomNinjaAtk** [comment]
I don't believe there is a way to perform targeted scans for Jellyfin/Emby. And because of that, you can just use Lidarrs built in notification system to accomplish the goal.

If you those apps provide targeted scan options and you can link to some documentation, I'll look into adding it. But I'm not aware that it's possible...

**Comment by PutzJimmy** [comment]
It does support [Jellyfin&Emby](https://github.com/Cloudbox/autoscan#targets)  but I'm no coder so i have no idea how it works :P

**Comment by PutzJimmy** [comment]
Here is where I found out about it https://www.youtube.com/watch?v=JYBVAzJBw2w
